apps/public-reference/src/typeorm-model-shim.js

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable */
+
+// columns
```

Comment:
I assume this file is essentially cut & pasted from [the shim provided by TypeORM](https://github.com/typeorm/typeorm/blob/master/extra/typeorm-model-shim.js). Do we want to host our own version of this, or should we literally use the shim provided by TypeORM? We could adjust the webpack config to look in `node_modules`:

```javascript
resolve: { // see: https://webpack.js.org/configuration/resolve/
  alias: {
    typeorm: path.resolve(__dirname, "../node_modules/typeorm/typeorm-model-shim")
  }
}
```

Reply:
I had to make one modification, but it was something they actually should have included in that file, so theoretically if it got upstreamed we could pull it from the module dir. But for now we need to keep it in-repo.

Reply:
@jaredcwhite Does [this PR](https://github.com/typeorm/typeorm/pull/6354) resolve the issue?

Reply:
@bencpeters Unfortunately not — the shim is missing `BaseEntity` for some reason, I had to add it in.